src/components/DeviceDetails/services/convertor/index.ts

```diff
@@ -0,0 +1,74 @@
+import APP_ROUTES from 'services/appRoutes';
+import { Device, DeviceDataItem } from '../..';
+
+interface DeviceData {
+  title: string;
+  props: DeviceDataItem[];
+  hasOwner: boolean;
+}
+
+const convertor = {
+  'devices': {
+    getData: (device: Device): DeviceData => {
+      const {
+        soort,
+        category,
+        organisation,
+        properties,
+      } = device || {};
+      const { Privacyverklaring } = properties || {};
```

Comment:
Can we please move this property back to te device? In the initial conversion of the data that is retrieved from the services there are for all type devices the device.privacy set. When this is not present, an empty string is used. So technicaly all devices are provided with a privacy property.

Reply:
gedaan